chunky/src/java/se/llbit/chunky/ui/render/tabs/GeneralTab.java

```diff
@@ -273,6 +275,10 @@ public GeneralTab() throws IOException {
     biomeColors.selectedProperty().addListener((observable, oldValue, newValue) -> {
       scene.setBiomeColorsEnabled(newValue);
     });
+    using3dBiomes.setTooltip(new Tooltip("Attempt to load biomes as 3d (Added in 1.18+)."));
```

Comment:
```suggestion
    using3dBiomes.setTooltip(new Tooltip("Attempt to load 3d biomes (added in 1.15)."));
```

Reply:
atm even `biomeColors` doesn't - should it? I can make them all follow the same behaviour

Reply:
What does biomeColors not do?

Reply:
replied to the wrong thing apparently - meant `Should this trigger the reload chunks dialog?`

Reply:
It probably should, but switching biome colors even requires reloading the chunks. Is there a dialog for that, too? Probably from the emitter sampling?